packages/react/jest.config.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
       // https://github.com/carbon-design-system/carbon-charts/issues/594
       functions: 69,
     },
+    './src/components/TimeSeriesCard/TimeSeriesCard.jsx': {
```

Comment:
This got re-added because I moved the functions to their own util file to make better readability